Resources/views/CRUD/edit_phpcr_one_association_script.html.twig

```diff
@@ -49,6 +49,7 @@ This code manage the one-to-many association field popup
                 }
                 jQuery('#sonata-ba-field-container-{{ id }}').trigger('sonata.add_element');
                 jQuery('#field_container_{{ id }}').trigger('sonata.add_element');
+                Admin.setup_icheck(jQuery('#field_container_{{ id }}'));
```

Comment:
Yes, I was thinking about that after I submitted the pull request. Maybe it would be even better to listen to the sonata.add_element event and do the reformatting stuff (iCheck and maybe Select2 also?) in a more generic way. Wasn't looking into that because I need a quick fix for the iCheck now